### PR TITLE
magento/devdocs#: Improvement - adding links

### PR DESCRIPTION
**Pull request by DenisSaltanahmedov**
## Purpose of this pull request

This pull request (PR) adding links to the architectural basics overview topic. 

## Affected DevDocs pages
-  https://devdocs.magento.com/guides/v2.3/architecture/archi_perspectives/ABasics_intro.html

## Links to Magento source code

-  https://github.com/magento/devdocs/blob/master/src/guides/v2.3/architecture/archi_perspectives/ABasics_intro.md

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/architecture/archi_perspectives/ABasics_intro.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ Magento incorporates the core architectural principles of object-oriented, PHP-b
...
 
-*  Magento technology stack
-*  Magento View Model
-*  Extensibility
-*  Modularity
+*  [Magento technology stack]({{page.baseurl}}/architecture/tech-stack.html)
+*  [Magento View Model]({{page.baseurl}}/extension-dev-guide/view-models.html)
+*  [Extensibility]({{page.baseurl}}/architecture/extensibility.html)
+*  [Modularity]({{page.baseurl}}/architecture/extensibility.html)
 *  Event-driven architecture
```
Comment (rogyar) [inline src/guides/v2.3/architecture/archi_perspectives/ABasics_intro.md]: We may use the following topic as a reference for the event-driven architecture.

https://devdocs.magento.com/guides/v2.3/extension-dev-guide/events-and-observers.html

File: src/guides/v2.3/architecture/archi_perspectives/ABasics_intro.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ Magento incorporates the core architectural principles of object-oriented, PHP-b
...
 The following discussion focuses on how these topics apply directly to Magento:
 
-*  Magento technology stack
-*  Magento View Model
-*  Extensibility
-*  Modularity
+*  [Magento technology stack]({{page.baseurl}}/architecture/tech-stack.html)
+*  [Magento View Model]({{page.baseurl}}/extension-dev-guide/view-models.html)
+*  [Extensibility]({{page.baseurl}}/architecture/extensibility.html)
+*  [Modularity]({{page.baseurl}}/architecture/extensibility.html)
```
Comment (rogyar) [inline src/guides/v2.3/architecture/archi_perspectives/ABasics_intro.md]: We may point modularity link to the exact section, so

https://devdocs.magento.com/guides/v2.3/architecture/extensibility.html#modularity

**Comment by dobooth** [comment]
running tests

**Pull request merged**